kernelcare/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "display_name": "kernelcare",
+  "maintainer": "schvaliuk@cloudlinux.com",
+  "manifest_version": "1.0.0",
+  "name": "kernelcare",
+  "metric_prefix": "kernelcare.",
+  "metric_to_check": "",
```

Include a metric that when submitted will mark the integration tile as `Installed`.
```suggestion
  "metric_to_check": "kernelcare.uptodate",
```

---

added